commands/credentials.js

```diff
@@ -11,22 +11,23 @@ function * run (context, heroku) {
   let showCredentials = co.wrap(function * () {
     const host = require('../lib/host')
     let addon = yield fetcher.addon(app, args.database)
-    let credentials = yield heroku.get(`/postgres/v0/databases/${addon.name}/credentials`,
-                                       { host: host(db) })
-    if no errors
+    try {
+      let credentials = yield heroku.get(`/postgres/v0/databases/${addon.name}/credentials`,
+                                       { host: host(addon) })
       cli.table(credentials, {
         columns: [
           {key: 'name', label: 'Credential'},
           {key: 'state', label: 'State'}
         ]
       })
-    else
+    } catch (err) {
+      if (err.statusCode !== 422) throw err
```

Comment:
422 feels weird here (since this is a GET and there is no body), but given this is temporary anyway, it's not worth nitpicking.